_episodes/02-file_parsing.md

```diff
@@ -42,7 +122,47 @@ This code opens a file for reading (that's the `r` part in the `open` function),
 {: .challenge}
 
 ## Searching for a pattern in your file
-The file we opened is an output file which calculates the energy (and a lot of other stuff!) for an ethanol molecule.  If you looked through the file in a text editor, you would see that the critical line says "Final Energy".  We want to search through this file and find that line.
+The file we opened is an output file which calculates the energy (and a lot of other stuff!) for an ethanol molecule.  As stated previously, the `readlines()` function put the file contents into a list where each element is a line of the file. You may remember from lesson 1 that a `for` loops can be used to execute the same code repeatedly. As we learned in the previous lesson, we can use a foor loop to iterate through elements in a list.
+
+Let's take a look at what's in the file.
+
+~~~
+for line in data:
+  print(line)
+~~~
+{: .language-python}
+
+This will print exactly what is in the file.
+
+If you through the output, you will see that the critical line says "Final Energy".  We want to search through this file and find that line, and print only that line. We can do this using an `if` statement.
```

Comment:
You are missing a word here.  If you look through the output or if you go through the output?